README.md

```diff
@@ -36,7 +37,7 @@ You should see the following:
 
 Run the following to give access to the user:
 
-    oc adm policy add-cluster-role-to-user system:master <username>
+    ./oc adm policy add-cluster-role-to-user system:master <username>
```

Comment:
You should not do this at all, you are trying to give admin "root" on the cluster and it is ddefinitely not necessary.
To give the admin user access to the project just run (assuming the user 'admin'):
./oc adm policy add-role-to-user admin admin